corehq/project_limits/models.py

```diff
@@ -97,3 +100,41 @@ class Meta:
     def __str__(self):
         domain = f"[{self.domain}] " if self.domain else ""
         return f"{domain}{self.key}: {self.limit}"
+
+    def save(self, *args, **kwargs):
+        super().save(*args, **kwargs)
+        self._get_global_limit.clear(self.__class__, self.key)
+        self._get_domain_specific_limit.clear(self.__class__, self.key, self.domain)
+
+    def delete(self, *args, **kwargs):
+        super().delete(*args, **kwargs)
+        self._get_global_limit.clear(self.__class__, self.key)
+        self._get_domain_specific_limit.clear(self.__class__, self.key, self.domain)
```

Comment:
I could be smarter about only clearing the cache for the specific methods when relevant (if `self.domain` is a blank string vs something specific)

Reply:
https://github.com/dimagi/commcare-hq/pull/35781/commits/ce7b0102aeada48e9af3adf4ea55d0dcd231604d

Reply:
...hmm but what someone modifies an existing limit to be scoped to domain (or vice versa). That would not be an ideal usage, but would result in us not clearing the correct cache with this updated logic...